code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -25,6 +25,44 @@
 	icon_state = "box"
 	item_state = "syringe_kit"
 	var/foldable = /obj/item/stack/material/cardboard	// BubbleWrap - if set, can be folded (when empty) into a sheet of cardboard
+	var/maxHealth = 20//health is already defined
+
+/obj/item/weapon/storage/box/New()
+	..()
+	health = maxHealth
+
+/obj/item/weapon/storage/box/proc/damage(var/severity)
+	health -= severity
+	check_health()
+
+/obj/item/weapon/storage/box/proc/check_health()
+	if (health <= 0)
+		Destroy()

```

Comment:
I've never seen `Destroy()` called on its own. Can you provide a reference for this being an actually valid play? I think it may run the risk of skipping some things that `qdel(src)` would otherwise arrange.